aiida/backends/tests/cmdline/commands/test_computer.py

```diff
@@ -554,6 +567,37 @@ def test_computer_list(self):
             # Something should be printed to stdout
             self.assertIsNotNone(result.output)
 
+    def test_computer_list_(self):
+        """Test the `--all` flag of  `verdi computer list`."""
+        name_configured = 'computer_configured'
+        name_unconfigured = 'computer_unconfigured'
+        name_disabled = 'computer_disabled'
+
+        computer_configured = self.create_new_computer(name_configured)
+        computer_unconfigured = self.create_new_computer(name_unconfigured)
+        computer_disabled = self.create_new_computer(name_disabled)
+
+        computer_configured.configure(self.user)
+        computer_disabled.configure(self.user)
+        computer_disabled.get_authinfo(self.user).enabled = False
+
+        # Without flags, only the configured and unconfigured computers should be shown
+        options = []
+        result = self.cli_runner.invoke(computer_list, options)
+        self.assertClickResultNoException(result)
+        self.assertIn(name_configured, result.output)
+        self.assertIn(name_unconfigured, result.output)
+        self.assertNotIn(name_disabled, result.output)
```

Comment:
I think one key point to test here is that computers set up/configured by other users don't appear in the list.

Reply:
Note that what user "setup" a computer is not stored. A `Computer` does not have a join on a `User` entry